src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -102,14 +102,19 @@ angular
         { data: data, showInLegend: false, marker: { enabled: false } },
         thresholdOptions
       )
-      thresholdSerie = @hc.addSeries(threshold)
-      Highcharts.addEvent(thresholdSerie, 'click', (_event)-> thresholdOptions.onClickEvent(thresholdSerie))
-      thresholdSerie
+      @hc.addSeries(threshold)
 
     removeThreshold: (kpiId)->
```

Comment:
same here: would it work if you were to use the series name instead of the kpi id?